src/main/resources/templates/db_scripts/schema.json

```
@@ -1033,9 +1033,14 @@
       "fromModuleVersion": "23.1.0"
     },
     {
-        "run":"after",
-        "snippetPath":"dropCallNumberNormalizationFunctions.sql",
-        "fromModuleVersion":"24.0.0"
+      "run": "after",
+      "snippetPath": "hridLock.sql",
+      "fromModuleVersion": "23.1.0"
```

This should be 24.0.0 now following other recent changes.
```suggestion
      "fromModuleVersion": "24.0.0"
```

---

Changed.